qa/httpCalls/wallet/dumpSecrets.py

```diff
@@ -4,10 +4,6 @@ def http_wallet_dumpSecrets(sidechainNode, file_path, api_key):
       j = {
         "path": file_path
       }
-      auth = {
-         "api_key": api_key
-      }
       request = json.dumps(j)
-      authHeader = json.dumps(auth)
-      response = sidechainNode.wallet_dumpSecrets(request, authHeader)
+      response = sidechainNode.wallet_dumpSecrets(request, api_key)
```

Comment:
I think, the method argument "api_key" can have defalut value "None".
In case of None we call just `sidechainNode.wallet_dumpSecrets(request)`
Same in other methods like this.
This will simplify the life for the user.

Reply:
Done